CSS/index.css

```diff
@@ -102,7 +102,7 @@
   transform: translate(-50%);
   background-color: white;
 }
-.article .closeButton {
+.article .close-button {
```

Comment:
Way to really make this code "yours" and follow a naming convention that works better for you.